extensions/wikia/ContributionAppreciation/ContributionAppreciationCommonHooks.class.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+class ContributionAppreciationCommonHooks {
+	const TRACKING_URL = 'https://beacon.wikia-services.com/__track/special/appreciation_email';
+	const EMAIL_CATEGORY = 'ContributionAppreciationMessage';
+
+	public static function onSendGridPostbackLogEvents( $events ) {
+		foreach ( $events as $event ) {
+			if ( static::isAppreciationEmailEvent( $event ) ) {
+				if ( preg_match( '/diff=([0-9]*)/', $event[ 'url' ], $diff ) ) {
+					static::sendDataToDW( 'button_clicked', $event[ 'wikia-email-city-id' ], $diff[ 1 ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikia-email-city-id" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)